src/libyang.h

```diff
@@ -131,6 +131,26 @@ extern "C" {
  * `next` pointer of the last item in the list is always NULL.
  */
 
+/**
+ * @page howtoThreads Threading Limitations
+ *
+ * @section context Context
+ *
+ * It is safe to read from ::ly_ctx structure concurrently and use its dictionary, which is protected by a lock.
+ * Thread-safe functions include any ones working with data trees (only context dictionary is accessed) and all
+ * the `ly_ctx_get_*()` functions. Generally, they are the functions with `const` context parameter.
+ *
+ * @section schema Schema Trees
```

Comment:
Not sure if there should be a separate section for schema trees. They are actually tightly connected with the context since schema modification can be actually done via context.

Reply:
They were not there originally, I myself was not sure whether it makes sense or not. So I have removed it.